src/JsCodeExtractor.php

```diff
@@ -28,6 +28,13 @@ public static function fromString( $string, Translations $translations, array $o
 		try {
 			$options += static::$options;
 
+			WP_CLI::debug(
```

Comment:
This takes up a lot of visual space for a debugging statement.

I'd prefer the following notation instead:
```php
WP_CLI::debug( "Parsing file {$options['file']}" );
```